src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.ts

```diff
@@ -65,4 +66,8 @@ export class HeaderExercisePageWithDetailsComponent implements OnInit, OnChanges
             this.icon = getIcon(exerciseType) as IconProp;
         }
     }
+
+    notReleased(): boolean {
```

Comment:
this method is not used, please delete it

Reply:
Done :-)